cunumeric/module.py

```diff
@@ -2100,11 +2099,36 @@ def repeat(a, repeats, axis=None):
                 category=UserWarning,
             )
         repeats = np.int64(repeats)
-        result = array._thunk.repeat(
-            repeats=repeats,
-            axis=axis,
-            scalar_repeats=True,
-        )
+        if repeats < 0:
+            return ValueError(
+                "'repeats' should not be negative: {}".format(repeats)
```

Comment:
Please use f-strings in new code

Reply:
and use `raise` to raise an exception instead of `return`